build.gradle

```diff
@@ -12,7 +12,7 @@
 apply from: file("${rootDir}/build-resources.gradle")
 allprojects {
     group = 'com.amazon'
-    version = '1.0.0'
+    version = '1.0.0.0-alpha2'
```

Comment:
I believe this should be `1.0.0.0-beta1`

Reply:
Resolved in offline discussion.